testFile.txt

```diff
@@ -0,0 +1,11 @@
+Line added in commit 1
+
+Line added in commit 2
+
+Line added in commit 3
+
+make some mess with commit 6
+
+Line added in commit 4 
```

Comment:
commit messages 4 and 5 could be combined, the are impementing the same functionality